src/vm/jitinterface.cpp

```diff
@@ -13345,12 +13345,13 @@ BOOL LoadDynamicInfoEntry(Module *currentModule,
 
     case ENCODE_VARARGS_METHODDEF:
         {
-            token = TokenFromRid(
-                        CorSigUncompressData(pBlob),
-                        mdtMethodDef);
+            mdSignature token = TokenFromRid(
+                                    CorSigUncompressData(pBlob),
+                                    mdtMethodDef);
 
             IfFailThrow(pInfoModule->GetMDImport()->GetSigOfMethodDef(token, &cSig, &pSig));
-
+        }
+        {
         VarArgs:
```

Comment:
@sandreenko btw. i was getting `note: jump bypasses variable initialization` here too because of `mdSignature token` but i was able to workaround it by starting a new scope, seems like clang is mad about bypassing variable initialization only if its in the same scope (even though the bypassed variable is dead at the point you jump to)

Reply:
Got it.